docs/guides/getting-started-orchestrate-apis.md

```diff
@@ -1,26 +1,47 @@
 ---
 id: orchestrate-apis
-title: Getting started with API orchestration
-sidebar_label: Getting started with API orchestration
+title: Get started with API orchestration
+sidebar_label: Get started with API orchestration
 description: "Use Connectors to build low code process automation solutions"
 keywords:
   [api endpoints, orchestration, getting started, user guide, connectors]
 ---
 
 <span class="badge badge--beginner">Beginner</span>
-<span class="badge badge--medium">Time estimate: 15 minutes</span>
+<span class="badge badge--medium">Time estimate: 15 minutes</span><br /><br />
 
 import clsx from "clsx";
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import SmPrereqs from './react-components/sm-prerequisites.md'
+import SaasPrereqs from './react-components/saas-prerequisites.md'
+import Install from './react-components/install-plain-java.md'
 
 This guide will walk you through working with a REST Connector task as a first time Camunda 8 user. The REST Connector is a [protocol Connector](/components/connectors/out-of-the-box-connectors/available-connectors-overview.md#protocol-connectors), where you can make a request to a REST API and use the response in the next steps of your process.
 
 :::note
 New to Connectors? Review our [introduction to Connectors](/components/connectors/introduction.md) to get familiar with their capabilities, and have a closer look at all of the available [out-of-the-box Connectors](/components/connectors/out-of-the-box-connectors/available-connectors-overview.md).
 :::
 
+<Tabs>
```

Comment:
Within the context of these changes, I'd considered something like:

```
Get started
- With Camunda Self-Managed
-- The java install info
-- the Spring guide
- With Camunda SaaS 
-- The account guide
-- Model your first process
- By use case
-- the existing use case docs
``` 

In a future iteration, something like this could naturally build out a more directed, result-oriented experience for the initial SaaS vs Self-Managed steps (where the setup is different, and then those personas might want/need slightly different initial onboarding experiences). We could then have more directed "Next steps" - other guides we commonly see those personas needing to head to next, which could be more unified/install agnostic use guides.